telemetry/metrics.yaml

```diff
@@ -1,24 +1,270 @@
----
-# Schema
 $schema: moz://mozilla.org/schemas/glean/metrics/2-0-0
 
-page:
-  load:
+moso_events: # If we're using a single metrics.yaml across Mozilla Social, what do I call the category? Can I omit a category?
+
+  # First we have some event properties that are re-used across all events
+  # I implemented these as uuid/string/quantity metrics, instead of nesting them as extra_keys inside events. Is this valid?
+
+  # Is it possible to attach them to the baseline ping, as I've done?
+
+  mastodon_account_id:
+    type: string
+    lifetime: application
+    description: >
+      The user's account ID from Mastodon.
+    bugs:
+      - TBD
+    data_reviews:
+      - TBD
+    data_sensitivity:
+      - technical
+    notification_emails:
+      - kdemtchouk@mozilla.com
+    expires: never
+    send_in_pings:
+      - events
+      - glean_internal_info
```

Comment:
I'm not sure which of the built-in ping types `glean_internal_info` corresponds to:
https://mozilla.github.io/glean/book/user/pings/sent-by-glean.html

Maybe we can drop this for now?

Reply:
I believe it comes through the `metrics` ping. Here's what it looks like when it comes through:

```
  "metrics": {
    "string": {
      "moso_events.mastodon_account_id": "111025579784690475",
      "moso_events.mastodon_instance_domain": "mozilla.social",
      "moso_events.mastodon_handle": "alytest01",
      "moso_events.page_url": "http://localhost:5314/discover",
      "moso_events.referrer_url": ""
    }
  },
  "events": [
    {
      "category": "moso_events",
      "name": "link_click",
      "timestamp": 0,
      "extra": {
        "target_url": "/home"
      }
    }
  ],
```

Reply:
@wtfluckey after doing this, i wonder if we need that. we should try.

Reply:
Yep, y'all are right - everything still works after removing `glean_internal_info`. Will make that change now!